src/main/java/sirius/biz/storage/layer2/Blob.java

```diff
@@ -187,7 +196,7 @@ public interface Blob {
      * Provides new content for this blob.
      * <p>
      * Also note that if a file is used to provide the new contents of this blob, use
-     * {@link #updateContent(String, File)} as this is likely way more efficient.
+     * {@link #updateContent(String, File)} as this way is likely more efficient.
```

Comment:
This was probably no typo, but a little informal. Strictly speaking, the original comment claims not just more efficiency, but considerably more efficiency. … Just sayin'

Reply:
Might be, but IntelliJ complained it was gramatically wrong